args.js

```diff
@@ -14,7 +14,7 @@ const optionsBook = {
         acceptedValues: range(1, 10000)
     },
     logpath: {
-        defaultValue: './activity.log',
+        defaultValue: `${__dirname}/activity.log`,
```

Comment:
Once we have some type of [automated windows tests running](https://github.com/plotly/plotly-database-connector/pull/64) these little cross-platform differences should get automatically caught 👌